README.md

```diff
@@ -40,14 +40,14 @@ SameBoy is an open-source project licensed under the MIT license, and you're wel
 SameBoy requires the following tools and libraries to build:
  * clang
  * make
- * Cocoa port: OS X SDK and Xcode command line tools
+ * Cocoa port: OS X SDK and Xcode command line tools [OSX Only]
```

Comment:
OS X already implied by the Cocoa, (and the requirement of the OS X SDK and Xcode), no need to put that here

Reply:
I didn't know that Cocoa was OSX-specific when I was first compiling, and when I saw the Windows-specific instructions below, it had implied that all of the above steps were general to all OS's.

I did think that Cocoa might have been OS-specific because I know what Xcode is, but the words "Cocoa port" made me think that there was a "port" of it for Windows. Might be good to break it apart into Linux/Mac/Windows sections to ensure no confusion?